src/components/atoms/SlideOut/SlideOut.tsx

```diff
@@ -33,20 +34,28 @@ export const SlideOut = ({ children, showCloseButton = true }: SlideOutProps) =>
 
   return (
     <AnimatePresence initial={false}>
-      {currentSlideOut === "concepts" && (
+      {currentSlideOut !== "" && (
```

Comment:
Praise: This could just be `currentSlideOut &&` (as empty string is falsy) but I prefer the explicit string comparison to give a sense of what this variable stores.